Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Creating-Saving-and-Publishing-Content/index.md

```diff
@@ -98,6 +98,48 @@ To set up scheduled unpublishing, follow these steps:
     ![Schedualed unpublishing.](images/Schedule_Unpublishing_v9.png)
 5. Select **Schedule**.
 
+## Comparing Content between environments
+
+:::note
+Content comparison is currently available in all Umbraco Cloud projects running Umbraco Deploy 9.3+ for Umbraco 9 and Umbraco Deploy 4.5+ for Umbraco 8.
+:::
+
+Compare Content allows to preview content changes before transferring them to another environment. This is helpful to ensure that the correct updates are transferred when working with content in multiple environments.
+
+You can see the **Summary Information** and **Field Comparision** values to understand what will change if you proceed to transfer the content upstream or try restoring content to the current environment.
```

Comment:
```suggestion
You can see the **Summary Information** and **Field Comparison** values to understand what will change if you proceed to transfer the content upstream or try restoring content to the current environment.
```

Reply:
Yes, I noticed the spelling is wrong, but that's how it's spelled in the UI and I kept the same as the image. Can we raise an issue to correct it?

Reply:
Ah, I see 🤔 

Yes. You can raise an issue here: https://github.com/umbraco/Umbraco-CMS/issues

Reply:
Raised issue: https://github.com/umbraco/Umbraco-CMS/issues/12352